### PR TITLE
Phpcs

### PR DESCRIPTION
**Pull request by jfeng-dev**
Added PHPCS to dev dependency,
added the phpcs command to travis,
added zumba coding standards to dev dependency

### CODE REVIEW

**Comment by young-steveo** [comment]
@j-feng13 The Zumba style guide does not use curly braces on a new line for classes, etc.  This looks like PSR-2.

Which, as a matter of fact, might be fine.  Since this is an open source project, maybe we want to use psr-2?

File: src/InvalidTimeStampResponse.php [hunk 1]
```diff
@@ -2,10 +2,19 @@
...
 namespace Zumba\VanillaJsConnect;
 
-class InvalidTimeStampResponse extends Response {
-
-	protected $error = 'invalid_request';
-
-	protected $message = 'The timestamp is invalid.';
+class InvalidTimeStampResponse extends Response
+{
+    /**
```
Comment (nkcmr) [inline src/InvalidTimeStampResponse.php:7]: @j-feng13 alignment issues.

File: src/Request.php [hunk 2]
```diff
@@ -2,40 +2,76 @@
...
+    public function getTimestamp()
+    {
+        return $this->timestamp;
+    }
+
+    /**
+         * Returns Request signature
+         *
+         * @return string
+         */
```
Comment (young-steveo) [inline src/Request.php]: @j-feng13 There are several of these doc-blocks that look a little funky.  Can you straighten them?

File: src/InvalidClientResponse.php [hunk 3]
```diff
@@ -2,20 +2,49 @@
...
+     */
+    private $error = 'invalid_client';
 
-	private $error = 'invalid_client';
-	private $clientID;
-	private $message = "Unknown client %s.";
+    /**
+     * ClientID that gets passed in for over riding toArray
+     *
+     * @var [type]
```
Comment (young-steveo) [inline src/InvalidClientResponse.php]: @j-feng13 [type]

**Pull request merged**